Remotion/ObjectBinding/Web/UI/Controls/BocDateTimeValueImplementation/Rendering/BocDateTimeValueRenderer.cs

```diff
@@ -241,7 +241,7 @@ private void RenderEditModeControls (BocDateTimeValueRenderingContext renderingC
       if (hasTimeField)
       {
         renderingContext.Writer.AddAttribute (
-            HtmlTextWriterAttribute.Class, CssClassTimeInputWrapper + " " + GetPositioningCssClass (renderingContext, DateTimeValuePart.Time));
+            HtmlTextWriterAttribute.Class, CssClassTimeInputWrapper + " " + CssClassDefinition.RemotionThemed + " " + GetPositioningCssClass (renderingContext, DateTimeValuePart.Time));
```

Comment:
If we define the property on the base renderer (see BocRendererBase), we can use the property here.